src/src/services/models/runs/runDetailAppModel.ts

```diff
@@ -146,7 +146,7 @@ function processRunBatchData(
       }),
       sortKey: `${run.name}${contextName}`,
     };
-    if (run.name.startsWith('__system__')) {
+    if (run.name.startsWith('system/')) {
```

Comment:
This is the third time I'm seeing this magic string in this PR - could we extract a constant somehow?

Reply:
This is based on the existing UI code from Aim at this stage. Extracting a constant would make total sense, but I wanted to keep this as close as possible to the current upstream logic.